local-volume/provisioner/deployment/kubernetes/helm/provisioner/values.yaml

```diff
@@ -0,0 +1,21 @@
+common:
+  namespace: default
+  defaultHostDir: /mnt/disks
```

Comment:
this is unused

Reply:
this is the default value I found in bootstrapper code. I can remove it from here completely or move it into the template.

Reply:
I mean, I don't see it used in the template anywhere.  If it's not used, then I would delete it